### ISSUE TITLE
COSMIC Development and Integration

### ISSUE THREAD

**Issue by castrojo** [ublue-os/main#534]
This issue tracks the integration of COSMIC into the rest of the ublue images. It's currently standalone.

**Comment by castrojo** [comment]
Ok so I think we can solve issues like this one by basing cosmic off of our base images instead of Fedora: https://github.com/ublue-os/cosmic/issues/14

This would bring in the right policy.json and all the QoL stuff that's expected out of the box. I think if we sort this and then enable the new ISO action by F40 then that would be a pretty good alpha. 

Most of the things in this image come via the RPM packaging so I think it'll be pretty low touch once we get it there. Thoughts? Also it'd be great if we could find more cosmic-interested folks to help out.

**Comment by ryanabx** [comment]
I think all of that sounds good

**Comment by castrojo** [comment]
Do you want me to PR in switching it to a ublue base image?

**Comment by ryanabx** [comment]
> Do you want me to PR in switching it to a ublue base image? 

Yes, shouldn't cause any issues to my knowledge!

**Comment by m2Giles** [comment]
We have switched over to using our images for 39 and 40 builds. We don't have a rawhide build so staying with upstream.

base, silverblue, and kinoite are all building.

We also turned off arm64 since I'm not exactly sure what those were even building since we don't have Arm builders.

**Comment by ryanabx** [comment]
Yes. In the future we should try to build all ublue images with arm, I don't know about the progress and blockers for that, but for now this is a fine change

**Comment by castrojo** [comment]
Still waiting on ARM builders in github. Tracking issue for the other stuff is here: https://github.com/ublue-os/main/pull/524
